src/legend/opacityLegendBehavior.ts

```diff
@@ -0,0 +1,58 @@
+/*
```

Comment:
This file should be listed in `tsconfig.json`

Reply:
It will be deleted